runtime/dart_vm_lifecycle.cc

```diff
@@ -13,7 +13,7 @@ namespace blink {
 // object weak pointer are behind this mutex.
 static std::mutex gVMMutex;
 static std::weak_ptr<DartVM> gVM FML_GUARDED_BY(gVMMutex);
-static std::shared_ptr<DartVM> gVMLeak FML_GUARDED_BY(gVMMutex);
+static std::shared_ptr<DartVM>* gVMLeak FML_GUARDED_BY(gVMMutex);
```

Comment:
This is the only difference between the previous patch and this reland. It used to be a shared pointer and now it is a pointer to one.